cc/txParser.js

```diff
@@ -116,17 +117,17 @@ const parseTransactionData = (tx) => {
     })
 
     // calculate change
-    let change = 0;
+    let change = new BN();
     if (changeReceivingAddress) {
       const txToAddress = tx.outs.find(s => s.address === changeReceivingAddress)
       if (txToAddress) {
-        change = txToAddress ? txToAddress.value : 0;
+        change = txToAddress ? new BN(txToAddress.value) : new BN();
```

Comment:
This should probably just be:

```js
if (txToAddress) {
  change = new BN(txToAddress.value);
}
```

Realize you were just following the previous pattern though.

Reply:
updated, thanks for catching it!